pkg/pki/ca/ca.go

```diff
@@ -22,15 +22,32 @@ import (
 	"errors"
 	"time"
 
+	"github.com/golang/glog"
 	"istio.io/auth/pkg/pki"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	apiv1 "k8s.io/client-go/pkg/api/v1"
 )
 
 const (
+	// IstioSecretType is the Istio secret annotation type.
```

Comment:
Not: all constants here can be unexported.

Reply:
They are used in ca_test.go

Reply:
That's in the sane package, right? Still don't need to export them.

Reply:
Same

Reply:
I see, just followed secret.go example, looks it used somewhere else.